src/ui/wxWidgets/PWSafeApp.cpp

```diff
@@ -272,9 +272,10 @@ bool PWSafeApp::OnInit()
   // For instance, the behavior of function 'wcstombs' depends on the LC_CTYPE 
   // category of the selected C locale.
 #if defined(__WXMAC__)
-  wxString typeFull;
-  if (wxGetEnv(wxT("LC_CTYPE"), &typeFull)) {
-    setlocale(LC_CTYPE, typeFull.c_str());
+  char *lcCType = setlocale(LC_CTYPE, NULL);
+
+  if (lcCType && strcmp(lcCType, "UTF-8")) { // MAC OS only have UTF-8 as default, but conversion with this string is nut running well
+    setlocale(LC_CTYPE, "");
   }
   else {
     setlocale(LC_CTYPE, "en_US.UTF-8");
```

Comment:
I wonder if "C" would be a better option as a fallback solution. Better in terms of portability if I look at the [man page](https://man7.org/linux/man-pages/man3/setlocale.3.html) of "setlocale". - Just a thought.